Makefile

```diff
@@ -218,7 +218,7 @@ test/instrumented-sample-app/certs/cert.pem test/instrumented-sample-app/certs/k
 .PHONY: test-e2e
 test-e2e: KUBECONFIG?=$(HOME)/.kube/config
 test-e2e: test/instrumented-sample-app/certs/cert.pem test/instrumented-sample-app/certs/key.pem
-	go test -timeout 55m -v ./test/e2e/ $(TEST_RUN_ARGS) --kubeconfig=$(KUBECONFIG) --operator-image=$(REPO):$(TAG) -count=1
+	go test -timeout 55m -v ./test/e2e/ $(TEST_RUN_ARGS) --kubeconfig=$(KUBECONFIG) --operator-image=$(REPO):master -count=1
```

Comment:
I use master tag here since the `Probe` feature is not released yet, so let's just use this to test in CI first. I can change the tag when this is included in the next release.

Reply:
This shouldn't need to be necessary, if you follow these [instructions](https://github.com/coreos/prometheus-operator#running-end-to-end-tests-on-local-kind-cluster) the tag based approach should work just fine.